frontend/js/src/settings/link-listens/LinkListens.tsx

```diff
@@ -198,10 +203,26 @@ export default function LinkListensPage() {
         <title>Link with MusicBrainz</title>
       </Helmet>
       <h2 className="page-title">Link with MusicBrainz</h2>
+      <ReactTooltip id="matching-tooltip" multiline>
+        We try our best to match every listen with a MusicBrainz recording in
+        order to enrich the information about the track, album and artist.
+        <br />
+        Sometimes the auto-matching fails to recognize tracks, but you can
+        manually link them below.
+        <br />
+        Other times, this data does not exist in MusicBrainz and needs to be
+        added.
```

Comment:
Suggestions:
"When a track can't be auto-matched you can manually link them here."
"Recordings may not exist in MusicBrainz, and need to be added there first."